src/SessionManager/ProgramSettings.cc

```diff
@@ -34,6 +38,141 @@ void applyOptionalArgument(T& val, const string& argument_name, const cxxopts::P
 }
 
 ProgramSettings::ProgramSettings(int argc, char** argv) {
+    if (argc > 1) {
+        spdlog::info("Using command-line settings");
+    }
+    ApplyCommandLineSettings(argc, argv);
+
+    const fs::path user_settings_path = fs::path(getenv("HOME")) / CARTA_USER_FOLDER_PREFIX / "backend.json";
+    const fs::path system_settings_path = "/etc/carta/backend.json";
+
+    json settings;
+    if (fs::exists(system_settings_path) && !no_system_config) {
+        settings = JSONConfigSettings(system_settings_path.string());
+        system_settings_json_exists = true;
+        spdlog::info("System settings found. Command-line arguments and will override system settings");
+        spdlog::info("If you want to ignore system settings, use --no_system_config as command-line argument.");
```

Comment:
These messages seem much too verbose for `info`. I would 

1. add a paragraph for the detailed precedence and option explanation to the program help message
2. change these just to print "Reading [system/user] settings from [path to file]"
3. change them to debug.